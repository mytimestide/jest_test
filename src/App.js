import React from 'react';
function App() {
  return (
    <div className="App" data-test="container" title="times">
      times!
    </div>
  );
}

export default App;
