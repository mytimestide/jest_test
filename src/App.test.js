import React from 'react';
import Enzyme, { shallow,mount } from 'enzyme'
import ReactDOM from 'react-dom'
import Adapter from 'enzyme-adapter-react-16'
import App from './App';

Enzyme.configure({adapter: new Adapter()})
test('renders learn react link', () => {
  // const div =document.createElement('div');
  // ReactDOM.render(<App />,div)
  // const container = div.getElementsByClassName('App')
  // expect(container.length).toBe(2)
  // const wrapper = shallow(<App />)
  // console.log(expect(wrapper.find('.App')));
  const wrapper = mount(<App />)
  expect(wrapper).toMatchSnapshot()
  // const container = wrapper.find('[data-test="container"]')
  // expect(container).toExist();
  // expect(container).toHaveProp('title','times');
});
