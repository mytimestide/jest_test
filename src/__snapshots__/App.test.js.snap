// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<App>
  <div
    className="App"
    data-test="container"
    title="times"
  >
    times!
  </div>
</App>
`;
